src/bdscript/channelType.md

```diff
@@ -6,7 +6,7 @@ Returns channel type.
 ```
 $channelType[channel ID]
 ```
-> The different channel types that the bot will retured are: `text`, `voice`, `category`, `thread`, `stage`, `announcement` and `forum`.
+> The different channel types that the bot will return are: `text`, `voice`, `category`, `thread`, `dm`, `stage`, `news` and `forum`.
```

Comment:
The News channel type does not exist
```suggestion
> The different channel types that the bot will return are: `text`, `voice`, `category`, `thread`, `dm`, `stage`, `announcement` and `forum`.
```

Reply:
Well BDFD return `announcement` as `news` tho

![Screenshot_20221101_003825](https://user-images.githubusercontent.com/95774950/199090082-b65500e1-2e47-4f8e-afb8-35deda8b5eba.png)

Reply:
It will change soon

Reply:
Oh! well...Thanks for the info.